flow/project.py

```diff
@@ -4172,18 +4175,22 @@ def foo(job):
         options : str
             A string to append to submissions. Can be any valid
             :meth:`FlowOperation.run` option. (Default value = "")
+        aggregator_obj : :class:`~.aggregator`
```

Comment:
Renamed to `group_aggregator` in 76c06048da93724d819de21e4c406241a668b38f, changed elsewhere for consistency and to avoid shadowing the imported name `aggregator`.